webpack.config.js

```diff
@@ -0,0 +1,78 @@
+const webpack = require('webpack');
+const path = require('path');
+
+// Config directories
+const SRC_DIR = path.resolve(__dirname, 'src');
+const OUTPUT_DIR = path.resolve(__dirname, 'dist');
```

Comment:
Is this leftover from the tutorial just scaffolding for the save/load tests feature?

Reply:
This is from the tutorial, however it is needed for the entry point for webpack (SRC: see line 13 ) and OUTPUT_DIR  is the path for the finalized bundles.

Reply:
Cool, thanks for clarifying